setup.py

```diff
@@ -143,6 +143,7 @@ def get_git_sha() -> str:
             "google-cloud-bigquery>=3.10.0",
         ],
         "clickhouse": ["clickhouse-connect>=0.5.14, <1.0"],
+        "databend": ["databend-sqlalchemy>=0.3.2, <0.5"],
```

Comment:
Is there some reason we want to support 0.3? Could we just update this to have the lower bound on the most recent verified version?

Reply:
The versions less than 0.3 have some compatibility issues. 
Of course, we can use this  lower bound. You mean we just `"databend": ["databend-sqlalchemy>=0.3.2"]`?

Reply:
@hantmac per [PyPI](https://pypi.org/project/databend-sqlalchemy/#history) the latest release of said package is `0.3.2` and thus the open upper limit of `0.5` seems somewhat arbitrary. Generally—assuming that the package is adhering to semver—placing an open upper limit of the major version (in this case `1.0`) is preferred as this _should_ prevent breaking changes.

Reply:
> @hantmac per [PyPI](https://pypi.org/project/databend-sqlalchemy/#history) the latest release of said package is `0.3.2` and thus the open upper limit of `0.5` seems somewhat arbitrary. Generally—assuming that the package is adhering to semver—placing an open upper limit of the major version (in this case `1.0`) is preferred as this _should_ prevent breaking changes.

You mean I should set `databend-sqlalchemy>=0.3.2, <1.0` ?